src/endpoints/getEvents.ts

```diff
@@ -0,0 +1,85 @@
+import { HLTVConfig } from '../config'
+import { fetchPage, toArray } from '../utils/mappers'
+import { EventResult } from 'models/EventResult';
+import { SimpleEvent } from 'models/SimpleEvent';
+import { EventSize } from 'enums/EventSize';
+
+export const getEvents = (config: HLTVConfig) => async ({
+    size
+}: {
+    size?: EventSize
+}): Promise<EventResult[]> => {
+    const $ = await fetchPage(`${config.hltvUrl}/events`, config.loadPage);
+
+    let events = [] as EventResult[];
+
+    toArray($('.events-month')).map(event => {
+        let monthEvents = [] as SimpleEvent[];
+        let monthName   = event.find('.standard-headline').text();
+
+        switch(size) {
+            case EventSize.Small:
+                monthEvents = parseEvents(toArray(event.find('a.small-event')), EventSize.Small);
+            break;
+
+            case EventSize.Big:
+                monthEvents = parseEvents(toArray(event.find('a.big-event')), EventSize.Big);
+            break;
+
+            default:
+                monthEvents = parseEvents(toArray(event.find('a.big-event'))).concat( parseEvents(toArray(event.find('a.small-event'))) );
+            break;
+        }
+
+        events.push({
+            month: monthName,
+            events: monthEvents
+        });
+    });
+
+    return events;
+}
+
+const parseEvents = (eventsToParse, size?:EventSize) => {
+    let events = [] as SimpleEvent[];
+
+    let dateSelector, nameSelector, locationSelector = '';
+
+    if(size == EventSize.Small) {
+        dateSelector        = '.eventDetails .col-desc span[data-unix]';
```

Comment:
Please use [Prettier](https://prettier.io/), all code in this library if formatted by it.

Reply:
Should be good now